src/main/java/com/cupid/jikting/member/service/FileUploadService.java

```diff
@@ -59,4 +65,27 @@ private String extractKeyFromImageUrl(String imageUrl) {
         UriComponents components = UriComponentsBuilder.fromUriString(imageUrl).build();
         return components.getPathSegments().get(components.getPathSegments().size() - 1);
     }
+
+    private void validIsFace(String fileName) {
+        AmazonRekognition rekognitionClient = AmazonRekognitionClientBuilder.defaultClient();
+        DetectFacesRequest request = new DetectFacesRequest()
+                .withImage(new Image()
+                        .withS3Object(new S3Object()
+                                .withName(fileName)
+                                .withBucket(bucket)))
+                .withAttributes(Attribute.DEFAULT);
+        try {
+            DetectFacesResult result = rekognitionClient.detectFaces(request);
+            List<FaceDetail> faceDetails = result.getFaceDetails();
+            if (faceDetails.isEmpty()) {
+                throw new BadRequestException(ApplicationError.PROFILE_IMAGE_NOT_FACE);
+            }
+            faceDetails.stream()
+                    .filter(face -> face.getConfidence() >= 90)
+                    .findFirst()
+                    .orElseThrow(() -> new BadRequestException(ApplicationError.PROFILE_IMAGE_NOT_FACE));
```

Comment:
p3. 불필요한 변수를 삭제하고 아래와 같이 작성하는건 어떨까요?

```suggestion
            AmazonRekognitionClientBuilder.defaultClient()
                    .detectFaces(request)
                    .getFaceDetails()
                    .stream()
                    .filter(face -> face.getConfidence() >= 90)
                    .findFirst()
                    .orElseThrow(() -> new BadRequestException(ApplicationError.PROFILE_IMAGE_NOT_FACE));
```